### PR TITLE
Branch add sort command

### PR DESCRIPTION
**Pull request by jylow**
Add sort command to code. #65 

Command enables users to sort the list by lexicographical ordering to enable users to more easily view names in the list.

At the moment, test cases have not been written. 

Further fields will be added to make use of different comparators to sort the list based on appointment times. Currently, the sort function is only to sort in ascending order and cannot be sorted back to its original order. This functionality works in tandem with the find function, causing the list to show sorted order of the finds if sort is called at anytime before the program is closed.

### CODE REVIEW

**Review by nikele2001 — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+    ---
```
Comment (sopa301) [inline docs/UserGuide.md:1]: This addition seems to significantly change the layout of the user guide. Could you confirm that it's intended?

File: src/test/java/seedu/address/logic/commands/AddCommandTest.java [hunk 2]
```diff
@@ -157,6 +158,12 @@ public ObservableList<Person> getFilteredPersonList() {
         public void updateFilteredPersonList(Predicate<Person> predicate) {
             throw new AssertionError("This method should not be called.");
         }
+
+
+        @Override
+        public void sortFilteredPersonList(Comparator<Person> comparator) {
+            throw new AssertionError("This method should not be called");
```
Comment (sopa301) [inline src/test/java/seedu/address/logic/commands/AddCommandTest.java:165]:
```suggestion
            throw new AssertionError("This method should not be called.");
```

**Review by sopa301 — COMMENTED** [review]
LGTM. A solution to the bug would be to modify list command to set sort predicate to a similar one like PREDICATE_SHOW_ALL_PERSONS in Model.java

File: docs/UserGuide.md (hunk 1 above)
Comment (jylow) [reply docs/UserGuide.md:1]: i didnt realise that i did this... can delete ah.

**Pull request merged**